inst/csv/replacementPatterns.csv

```diff
@@ -247,4 +247,50 @@ From,To,Pattern,Replacement
 "sql server","impala",".dbo.","."
 "sql server","impala","##","<hash><hash>"
 "sql server","impala","#","%temp_prefix%%session_id%"
-"sql server","impala","<hash><hash>","##"
\ No newline at end of file
+"sql server","impala","<hash><hash>","##"

```

Comment:
Oops, yes, you're right. @gowthamrao , ignore my remark. It was about the Impala rows